iroha/src/kura.rs

```diff
@@ -260,7 +266,11 @@ mod tests {
             .validate(&WorldStateView::new(World::new()), &AllowAll.into())
             .sign(&keypair)
             .expect("Failed to sign blocks.");
-        assert!(BlockStore::new(dir.path()).write(&block).await.is_ok());
+        assert!(BlockStore::new(dir.path())
+            .unwrap()
```

Comment:
Please use expect in tests

Reply:
I guess it won't make difference, as unwrap will show a line of code which panicked. I would personally spend time on tests themselves and their structure, rather than on writing comments in unwraps.

To me tests should be quick and dirty. You should cover only good cases, as they are more likely. Another property for them is to be easy to follow, but expects are just giving some unnecessary context to failing cases for the reader (which are unlikely to happen anyway).

Reply:
Unwraps are usually with ugly errors it hinders the ability to understand what went wrong in a test. Also they are a pain when started in a separate task or thread.

Reply:
I guess it won't make much difference, as they both panics. Report is ugly, but it is enough for devs to have a line of code.

Reply:
Ok